main.go

```diff
@@ -44,6 +48,27 @@ func createApp() *cli.App {
 	return app
 }
 
+func addBefore(app *cli.App, debug *bool) {
+	app.Before = func(c *cli.Context) error {
+		if *debug {
+			log.SetLevel(log.DebugLevel)
+		}
+		return nil
+	}
+}
+
+// addFlags - Add global flags to CLI application
+func addFlags(app *cli.App) {
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:        "debug, d",
+			Usage:       "switch on debug log output",
+			EnvVar:      "KORP_LOG_DEBUG",
+			Destination: &debug,
```

Comment:
without the global variable, this line can be removed

Reply:
I agree.